### DISCUSSION TITLE
how to set gmx_MMPBSA parameter of mctrdz

### DISCUSSION THREAD

**Discussion by sf-polaris** [Valdes-Tresanco-MS/gmx_MMPBSA#436] [Q&A] [answered]
Hi , I want to know  how to set gmx_MMPBSA parameter of mctrdz, I want to calculate the binding energy of ligands bound the GPCR ( membrane protein)  using gmx_MMPBSA, but **I don't know what the parameter of "mctrdz" means**. 
<img width="283" alt="屏幕截图 2023-11-29 203327" src="https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/34822b65-7231-47e0-816f-38567f664db3">

**Comment by marioernestovaldes** [comment, marked as answer]
mctrdz (Default = 0.0) is essentially the membrane center (in Å) in the z-direction. In a system centered at the origin (x=0, y=0, z=0) mctrdz is equal to 0. However, this is not the case for most systems hence the need for this parameter. Basically, you can open the input structure, choose an atom that would be located roughly in the middle of the membrane, and use its z-coordinate as mctrdz.

**Reply by sf-polaris** [reply]
Thank you for your replay！I choose a residue as the center of the membrane and I have tested a lot of central values，as shown in blow because I want to know the effect of different location of membrane on binding energy.  
<img width="152" alt="屏幕截图 2023-11-30 091312" src="https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/assets/151926066/1a808e43-5302-4d90-b4a6-8758f030c616">
As the figure shown, the center of the membrane is "63". I set the mthick parameter to 40. And the height of the protein is 60. 
When the central values vary from 0 to 63,  the binding energies are similar. But when the central values change to 120 , the binding energy has a big change.

**Comment by marioernestovaldes** [comment]
The reason for the `mctrdz` and `mthick` parameters is to consider implicit membrane conditions in roughly the same region where the actual explicit membrane would be. It's possible that with mctrdz=120 the implicit membrane conditions are far beyond what they should be, hence the change in binding free energy.

**Reply by sf-polaris** [reply]
OK, thank you!  I think the mctrdz=0 is similar to the mctrdz=120 because they are all 60 away from the center. Why mctrdz=0 is extremely similar to mctrdz=63, but mctrdz=120 is not ?
I also have another  two questions.
1、The  implicit membrane can influence the dielectric constant of the ligand binding pocket. If the binding pocket can go through water, I have to set the **poretype=1** , and when I set the membrane region around the binding pocket , whether the **dielectric constant of the binding pocket changes to the membrane's dielectric constant**? And whether the binding pocket  actually **has two dielectric constants——the water's dielectric constant and membrane's dielectric constant** ?
2、Whether the gmx_MMPBSA considers the **boundary conditions** when calculating the binding energy?
